src/backend/columnar/cstore_tableam.c

```diff
@@ -677,7 +678,7 @@ static void
 LogRelationStats(Relation rel, int elevel)
```

Comment:
function is missing. a comment when it is called. hard to review what we are adding output to. I guess this is `VACUUM VERBOSE`?